frontend/src/metabase/nav/components/WhatsNewNotification/WhatsNewNotification.styled.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from "@emotion/styled";
+import IconButtonWrapper from "metabase/components/IconButtonWrapper";
+import { color } from "metabase/lib/colors";
+
+export const DismissIconButtonWrapper = styled(IconButtonWrapper)`
+  color: ${color("bg-dark")};
```

Comment:
Now that we have the Mantine colors maybe you can use those - basically you can access them by using:

```
${ ({theme}) => {
    return css`
        color: ${theme.colors.bg[2]}
        
        &:hover: {
            color: ${theme.colors.text[1]}
        }
    `
}}
```

Reply:
The `theme` object is now injected into all of our styled components, so we always have access to that 😄